File: Fgts.c
#include <stdio.h>

int main(){
    float fgts = 0.0;
    float salario = 0.0;
    printf("digite seu salario: ");
    scanf("%f", &salario);
    fgts = salario * 0.08;
    printf("Valor do FGTS: %.2f \n", fgts);
    return 0;
}

File: pythonTeste/fgts.py
def calcular_fgts(salario_bruto: float) -> float:
    """
    Calcula o FGTS com base em 8% do salário bruto.
    """
    if salario_bruto < 0:
        raise ValueError("O salário bruto não pode ser negativo.")
    fgts = salario_bruto * 0.08
    return fgts

# Exemplo de uso
try:
    salario = float(input("Digite o salário bruto: R$ "))
    valor_fgts = calcular_fgts(salario)
    print(f"Valor do FGTS: R$ {valor_fgts:.2f}")
except ValueError as e:
    print(f"Erro: {e}")
